### PR TITLE
Removing Event Bus

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

File: src/OrchardCore.Modules/Orchard.Setup/Services/SetupService.cs [hunk 1]
```diff
@@ -207,8 +208,10 @@ await scope
                     };
 
                     // Invoke modules to react to the setup event
-                    var eventBus = scope.ServiceProvider.GetService<IEventBus>();
-                    await eventBus.NotifyAsync<ISetupEventHandler>(x => x.Setup(
+                    var setupEventHandlers = scope.ServiceProvider.GetRequiredService<IEnumerable<ISetupEventHandler>>();
+                    var logger = scope.ServiceProvider.GetRequiredService<ILogger<SetupService>>();
```
Comment (Jetski5822) [inline src/OrchardCore.Modules/Orchard.Setup/Services/SetupService.cs]: GetRequiredServices, then the Ienumerable is implied

**Comment by Jetski5822** [comment]
Really nicely done. I wonder what the perm and memory diff will be, before and after.

File: src/OrchardCore.Modules/Orchard.Setup/Services/SetupService.cs (hunk 1 above)
Comment (sebastienros) [reply src/OrchardCore.Modules/Orchard.Setup/Services/SetupService.cs]: Right, I think I saw that after, will change.

**Comment by sebastienros** [comment]
updated.

**Pull request merged**